pt-br/ruby-pt.html.markdown

```diff
@@ -4,8 +4,21 @@ lang: pt-br
 filename: learnruby-pt.rb
 contributors:
   - ["Bruno Henrique - Garu", "http://garulab.com"]
+  - ["David Underwood", "http://theflyingdeveloper.com"]
+  - ["Joel Walden", "http://joelwalden.net"]
+  - ["Luke Holder", "http://twitter.com/lukeholder"]
```

Comment:
I thought I should follow the "original" in English , and update with contributors from there, sorry =/